### ISSUE TITLE
Using mechanical hard drives, tasks often stop

### ISSUE THREAD

**Issue by bi-kai** [Chia-Network/chia-blockchain#3737]
![image](https://user-images.githubusercontent.com/15070379/117083733-f2229c00-ad77-11eb-8052-53359cfc5e28.png)

client logs
```
	Bucket 105 uniform sort. Ram: 3.251GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
	Bucket 106 uniform sort. Ram: 3.251GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
	Bucket 107 uniform sort. Ram: 3.251GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
	Bucket 108 uniform sort. Ram: 3.251GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
	Bucket 109 uniform sort. Ram: 3.251GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.
	Bucket 110 uniform sort. Ram: 3.251GiB, u_sort min: 0.563GiB, qs min: 0.281GiB.
Only read 482 of 262143 bytes at offset 126352926 from "X:\\temp\\plot-k32-2021-05-05-07-00-97c7f9e1926282f3390e97b4e2c1f39c87ea71c014edf1cb4208d6205bd7db56.plot.p1.t1.sort_bucket_110.tmp"with length 301948407. Error 1. Retrying in five minutes.
Only read 0 of 262143 bytes at offset 126352926 from "X:\\temp\\plot-k32-2021-05-05-07-00-97c7f9e1926282f3390e97b4e2c1f39c87ea71c014edf1cb4208d6205bd7db56.plot.p1.t1.sort_bucket_110.tmp"with length 301948407. Error 1. Retrying in five minutes.
Only read 0 of 262143 bytes at offset 126352926 from "X:\\temp\\plot-k32-2021-05-05-07-00-97c7f9e1926282f3390e97b4e2c1f39c87ea71c014edf1cb4208d6205bd7db56.plot.p1.t1.sort_bucket_110.tmp"with length 301948407. Error 1. Retrying in five minutes.
Only read 0 of 262143 bytes at offset 126352926 from "X:\\temp\\plot-k32-2021-05-05-07-00-97c7f9e1926282f3390e97b4e2c1f39c87ea71c014edf1cb4208d6205bd7db56.plot.p1.t1.sort_bucket_110.tmp"with length 301948407. Error 1. Retrying in five minutes.
Only read 0 of 262143 bytes at offset 126352926 from "X:\\temp\\plot-k32-2021-05-05-07-00-97c7f9e1926282f3390e97b4e2c1f39c87ea71c014edf1cb4208d6205bd7db56.plot.p1.t1.sort_bucket_110.tmp"with length 301948407. Error 1. Retrying in five minutes.
```

disk space is enought:
![image](https://user-images.githubusercontent.com/15070379/117083891-504f7f00-ad78-11eb-838c-f9eda0d15ee0.png)

system: win10
client: 1.1.2
---------------------------------------------------------------------------------------------------------------------------
start a new task, stoped again:
```
2021-05-05T08:11:04.158  chia.plotting.create_plots       : [32mINFO    [0m Creating 1 plots of size 32, pool public key:  8d08eec041f23a6ea822f8e458b87ab0e36da6fa56c724d77a5e3bfc477f94f00e5607069566477c7629ee3199871ff4 farmer public key: 92adc7ff6f313bb4d235bd47efd2889319cc1f825d015e639a138bec4a0ec6d7835e9b7a73a33d33134241347a098b01[0m
2021-05-05T08:11:04.178  chia.plotting.create_plots       : [32mINFO    [0m Memo: 8d08eec041f23a6ea822f8e458b87ab0e36da6fa56c724d77a5e3bfc477f94f00e5607069566477c7629ee3199871ff492adc7ff6f313bb4d235bd47efd2889319cc1f825d015e639a138bec4a0ec6d7835e9b7a73a33d33134241347a098b0170f4207ce7e4c6d05cc6b707d49dbb0395d3b260de19fbb2104c70c03fcdfca4[0m
2021-05-05T08:11:04.180  chia.plotting.create_plots       : [32mINFO    [0m Starting plot 1/1[0m

Starting plotting progress into temporary dirs: X:\temp2 and X:\temp2
ID: c96ed6b9e097edce29096525e7642c156bb2d0884ab38aff74dd3dc3ffd492da
Plot size is: 32
Buffer size is: 3390MiB
Using 128 buckets
Using 2 threads of stripe size 65536

Starting phase 1/4: Forward Propagation into tmp files... Wed May  5 08:11:04 2021
Computing table 1
Caught plotting error: Could not open X:\temp2\plot-k32-2021-05-05-08-11-c96ed6b9e097edce29096525e7642c156bb2d0884ab38aff74dd3dc3ffd492da.plot.p1.t1.sort_bucket_090.tmp: Invalid argument.
[10872] Failed to execute script chia
Traceback (most recent call last):
  File "chia\cmds\chia.py", line 81, in <module>
  File "chia\cmds\chia.py", line 77, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\plots.py", line 135, in create_cmd
  File "chia\plotting\create_plots.py", line 176, in create_plots
RuntimeError: Unknown exception
```

task with ssd is ok, but filed with hdd.

**Comment by dorkmo** [comment]
you might be running out of ram

**Comment by Evil0ctal** [comment]
I have the same issue dont use raid,that will get ploting stuck,it may be the chia core Bug.

**Comment by bi-kai** [comment]
![image](https://user-images.githubusercontent.com/15070379/117085437-547d9b80-ad7c-11eb-9524-bfcc5291192b.png)
my disk cant be readable now.
![image](https://user-images.githubusercontent.com/15070379/117085471-770fb480-ad7c-11eb-956c-353aac58b715.png)
maybe some error with disk, but dont sure is it caused by chia.

**Comment by bi-kai** [comment]
> you might be running out of ram

![image](https://user-images.githubusercontent.com/15070379/117085516-9d355480-ad7c-11eb-9034-d9dee2bf931d.png)
mem is enought.

**Comment by MikhailZelenkov** [comment]
Replace RAM ;)

**Comment by bi-kai** [comment]
> Replace RAM ;)

@MikhailZelenkov ram seems ok.
![image](https://user-images.githubusercontent.com/15070379/117086141-547e9b00-ad7e-11eb-9f42-d594841f5422.png)

other 14 tasks with ssd are correct:
![image](https://user-images.githubusercontent.com/15070379/117086187-7c6dfe80-ad7e-11eb-8649-e5a471c4ec98.png)

**Comment by Plopsi** [comment]
anti-virus blocking file access?

The disk down could be, chia is expensive on TBW, normally you should be able to plot 600 plots on a single 1000TBW disk before it starts to fail, but you know, from time to time there are mondays - even in products and production :D - check the SMART status of the disk.

**Comment by L0rDAst3r1X** [comment]
I'm having the same issue but it is only with one disk, I have 4 Plots in parallel in a different disk  and only  one fails, I thought that was RAM but the other ones are ok, even assign more ram to this one and failed, crystal disk info doesn't give any alert, the alerted one ends the plots flawlessly, I'm beginning to think that its something related with space even though it has 420GB free, or on how I formatted it I used the 2048 KB allocation unit size  because that gives me 10% more speed in tests, I will do some spece regardin tests and allocation unit size to see if it works and keep ypou posted

**Comment by ademnatural** [comment]
I'm having the same issue on Chia blockhain 1.1.6
any can help?

**Comment by darderik** [comment]
Same problem on 1.1.6, with a raid 0 volume. Disks are ok, ram is band new, and the file is just there at the path specified by the plotter output. Unfortunately not an hardware issue
